includes/specials/SMW_SpecialAsk.php

```diff
@@ -489,13 +489,6 @@ protected function getInputForm( $printoutstring, $urltail, $navigation = '', $d
 			// Other options fieldset
 			$result .= '<fieldset id="options" class="smw-ask-options-fields"><legend>' . wfMessage( 'smw_ask_otheroptions' )->escaped() . "</legend>\n";
 
-			// Info text for when the fieldset is collapsed
-			$result .= Html::element( 'div', array(
-				'class' => 'collapsed-info',
-				'style' => 'display:none;'
-				), wfMessage( 'smw-ask-otheroptions-collapsed-info')->text()
-			);
-
```

Comment:
@mwjames I believe this used to be "smw-prefs-ask-options-collapsed-default" even for 2.5x but was changed to  "smw-ask-otheroptions-collapsed-info" in 3.0.x https://github.com/SemanticMediaWiki/SemanticMediaWiki/commit/889918b2657c17e74dea97c2252184ca0708200e#diff-293fce79763cffb3923b9b1cc6f9f66c. So I just documented [link this](https://www.semantic-mediawiki.org/w/index.php?title=Help%3AUser_preferences&type=revision&diff=55959&oldid=55580)